### PR TITLE
418 resolve light warnings for mylyn builds

### PR DESCRIPTION
**Pull request by gnl42**
If anyone can figure out how to fix the complaints about AbstractBuildEditorPart I would love to see the solution

### CODE REVIEW

File: mylyn.builds/org.eclipse.mylyn.builds.ui/src/org/eclipse/mylyn/internal/builds/ui/editor/BuildOutputPart.java [hunk 1]
```diff
@@ -37,7 +38,7 @@ public class BuildOutputPart extends AbstractBuildEditorPart {
 
 	public BuildOutputPart() {
 		super(ExpandableComposite.TITLE_BAR | ExpandableComposite.EXPANDED);
-		setPartName("Output");
+		setPartName("Output"); //$NON-NLS-1$
```
Comment (ruspl-afed) [inline mylyn.builds/org.eclipse.mylyn.builds.ui/src/org/eclipse/mylyn/internal/builds/ui/editor/BuildOutputPart.java:41]: this should be externalized

File: org.eclipse.mylyn.releng/launcher/mylyn verify.launch [hunk 2]
```diff
@@ -13,7 +13,7 @@
         <listEntry value="dash.fail=false"/>
     </listAttribute>
     <stringAttribute key="M2_RUNTIME" value="EMBEDDED"/>
-    <booleanAttribute key="M2_SKIP_TESTS" value="false"/>
+    <booleanAttribute key="M2_SKIP_TESTS" value="true"/>
```
Comment (ruspl-afed) [inline org.eclipse.mylyn.releng/launcher/mylyn verify.launch:16]: Perhaps, this is unrelated

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
One UI string to be externalized

File: mylyn.builds/org.eclipse.mylyn.builds.ui/src/org/eclipse/mylyn/internal/builds/ui/editor/BuildOutputPart.java (hunk 1 above)
Comment (gnl42) [reply mylyn.builds/org.eclipse.mylyn.builds.ui/src/org/eclipse/mylyn/internal/builds/ui/editor/BuildOutputPart.java]: Everything has been externalized

File: org.eclipse.mylyn.releng/launcher/mylyn verify.launch (hunk 2 above)
Comment (gnl42) [reply org.eclipse.mylyn.releng/launcher/mylyn verify.launch]: Yeah, I keep switching between no tests/multi threaded. 
The files keeps getting caught with everything else. 
Need a way of having personalized launch files somehow

**Comment by ruspl-afed** [comment]
> If anyone can figure out how to fix the complaints about AbstractBuildEditorPart I would love to see the solution

This is not trivial, since `org.eclipse.emf.databinding.IEMFValueProperty` ignores type argument from inherited `org.eclipse.core.databinding.property.value.IValueProperty`

@merks do you know regarding any plans to update "org.eclipse.emf.databinding"?

**Comment by merks** [comment]
> do you know regarding any plans to update "org.eclipse.emf.databinding

There are no plans for that.
